### PR TITLE
Update eventing and cloudevents SDK.

### PR DESCRIPTION
**Pull request by ian-mi**
## Proposed Changes

  * Update eventing and events SDK.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -137,7 +137,7 @@ required = [
 
 [[constraint]]
   name = "github.com/cloudevents/sdk-go"
-  version = "1.0.0"
+  revision = "427e8cf75237fdc7d4bb6c496398d7288efb8921"
```
Comment (matzew) [inline Gopkg.toml]: @n3wscott   I guess, based on 

https://github.com/knative/eventing/pull/2589#pullrequestreview-360121978


this is than ultimately required, right ?

Comment (n3wscott) [reply Gopkg.toml]: until there is a release, yeah this is required.

**Pull request merged**